src/DateInput/DayInput.jsx

```diff
@@ -64,8 +64,8 @@ export default class DayInput extends PureComponent {
         onChange={onChange}
         onFocus={event => select(event.target)}
         onKeyDown={onKeyDown}
-        onKeyUp={event => updateInputWidth(event.target)}
-        placeholder="--"
+        onKeyUp={event => appendInputValue(event, onChange)}
```

Comment:
Hmm, so now this function does not run on key up? I think this improved input behavior 🤔